controllers/bindings/client_factory.go

```diff
@@ -186,6 +188,11 @@ func (cf *CachingClientFactory) GetClient(ctx context.Context, currentNamespace
 			ttl = cf.MaxClientCacheTTL
 		}
 
+		_, err = cl.(client.Client).RESTMapper().ResourcesFor(schema.GroupVersionResource{Version: "v1"})
```

Comment:
Is there a cheaper call we could use here? There are 44 resources in `v1` and we don't use the result at all. Maybe a nitpick, because this returns a quite compact object but it is nevertheless 44 times larger than we need :)

Reply:
Or we can leave it as is if no other call is an obvious choice...